pyproject.toml

```diff
@@ -22,6 +21,8 @@ dependencies = [
 [project.scripts]
 meds_etl_mimic = "meds_etl.mimic:main"
 meds_etl_omop = "meds_etl.omop:main"
+meds_etl_flat = "meds_etl.flat:convert_flat_to_meds_main"
```

Comment:
It looks like there are mismatches between the scripts and the function calls. I'm not sure whether it's me or the code that's muddled.

I would expect:
- `meds_etl_flat` to convert `meds` to `meds-flat`. 
- `meds_reverse_etl_flat` to convert from `meds-flat` to `meds`.

Here, `meds_etl_flat` calls `convert_flat_to_meds_main` which is the opposite of what I'd expect (and `meds_reverse_etl_flat` calls `convert_meds_to_flat_main`, also opposite.)